.github/workflows/cicd.yaml

```
@@ -13,84 +13,81 @@ on:
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v4
 
-    - name: Set up Go 1.22
-      uses: actions/setup-go@v2
-      with:
-        go-version: 1.22
+      - name: Set up Go 1.22
+        uses: actions/setup-go@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-go@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)